frontend/packages/dev-console/src/components/pipelines/__tests__/PipelineDetailsPage.spec.tsx

```diff
@@ -0,0 +1,31 @@
+import * as React from 'react';
+import { shallow } from 'enzyme';
+import { referenceForModel } from '@console/internal/module/k8s';
+import { PipelineModel } from '../../../models';
+import PipelineDetailsPage from '../PipelineDetailsPage';
+
+const pipelineDetailsProps: React.ComponentProps<typeof PipelineDetailsPage> = {
+  match: {
+    isExact: true,
+    path: `/k8s/ns/:ns/${referenceForModel(PipelineModel)}/:name/`,
+    url: 'k8s/ns/tekton-pipelines/tekton.dev~v1alpha1~Pipeline/simple-pipeline',
+    params: {
+      ns: 'tekton-pipelines',
+    },
+  },
+  kind: 'Pipeline',
+};
+
+const expectedTabs: string[] = [`Overview`, `YAML`, `Pipeline Runs`, `Parameters`, `Resources`];
+
+describe('Pipeline details page tests', () => {
+  it('should render `DetailsPage` Component with proper page tabs -> [ `Overview` , `YAML` , `Pipeline Runs`, `Parameters`, `Resources` ]', () => {
```

Comment:
I think you can keep the title of the test case short:
```suggestion
  it('should render `DetailsPage` Component with proper page tabs', () => {
```

Reply:
I agree, please remove the tab names from the test name.